pkg/controller/migplan/migplan_controller.go

```diff
@@ -648,12 +656,26 @@ func (r ReconcileMigPlan) generatePVResizeConditions(pvResizingRequiredVolumes [
 				Category: Warn,
 				Reason:   Done,
 				Message: fmt.Sprintf(
-					"Capacity of the following volumes will be automatically adjusted to avoid disk capacity issues in the target cluster  :  [%s]",
+					"Capacity of the following volumes will be automatically adjusted to avoid disk capacity issues in the target cluster:  [%s]",
 					strings.Join(pvResizingRequiredVolumes, ",")),
 				Durable: true,
 			})
 		}
 	} else {
 		plan.Status.DeleteCondition(PvCapacityAdjustmentRequired)
 	}
+	// PV analysis did not fail, but the pv resizing information couldn't be found for some volumes
+	// this indicates that MigAnalytic skipped these volumes for some other reason
+	if len(pvResizingMissingVolumes) > 0 {
+		plan.Status.SetCondition(migapi.Condition{
+			Type:     PvUsageAnalysisFailed,
+			Status:   True,
+			Category: Warn,
+			Reason:   NotDone,
+			Message: fmt.Sprintf(
+				"Failed to compute PV resizing data for the following volumes. Please ensure that the volumes are attached to one or more running Pods: [%s]",
```

Comment:
@pranavgaikwad this message is a good start, I wonder if we could go further and tell the user what the consequence will be if they just proceed without setting up a running Pod.

E.g., does this mean that `PVC migration may fail if actual storage usage exceeds requested storage.`